language/diem-vm/src/diem_vm.rs

```diff
@@ -516,27 +515,6 @@ pub fn txn_effects_to_writeset_and_events_cached<C: AccessPathCache>(
     Ok((ws, events))
 }
 
-pub(crate) fn charge_global_write_gas_usage<R: RemoteCache>(
```

Comment:
I'm not sure if I quite follow why this is going away in this PR? My understanding of the change was that it should only be fixing the logic in `session.num_mutated_accounts` to only return the number of accounts that were mutated, and not include the read/non-mutated accounts in the session?